apps/bajour/components/bajour/best-of-wepublish/best-of-wepublish.tsx

```diff
@@ -0,0 +1,131 @@
+import {styled} from '@mui/material'
+import {ApiV1, BuilderTeaserGridBlockProps} from '@wepublish/website'
+
+import {BestOfWePublishTeasers} from './best-of-wepublish-teasers'
+
+export type BestOfWePublishProps = Omit<BuilderTeaserGridBlockProps, 'teasers'> & {
```

Comment:
```suggestion
export type BestOfWePublishProps = BuilderTeaserGridBlockProps & {
```

Reply:
why this change? because it causes some problems with typings when I do that

Reply:
I don't have those issues. That change is as there's no need to omit it and then override it.